src/components/ThreejsEditorModal.jsx

```diff
@@ -117,15 +139,20 @@ export class ThreejsEditorModal extends ModalDialog {
         const toolbar = new Toolbar(this.editor);
         this.domElement.appendChild(toolbar.dom);
 
-        this.initializeControls();
-
+        this.initializeControlsInEditor();
         this.initializeLights();
     }
 
     /**
      * Add dragover listeners to group the objects.
      */
     addEventListeners() {
+        // TODO: this is a trick to disable orbit controls when multiple selection is active.
+        const isMultipleSelectionActive = () => {
+            const toggleButton = document.querySelector(".ms-button__toggle");
```

Comment:
Seems a bit hacky - what is `.ms-button__toggle` and where does it come from?

Reply:
It is a hack. Comes from the button for toggling Multiple selections:
![image](https://github.com/Exabyte-io/wave.js/assets/79542055/cc4b9752-f003-4492-8cdf-956b8eb656e1)

I couldn't find another solution in reasonable amount of time. It should work till the complete revision of 3D editor, where we'll be able to control every action and signal.